x-pack/plugins/secops/server/init_server.ts

```diff
@@ -36,6 +36,7 @@ export const initServer = (libs: AppBackendLibs, config: Config) => {
       createSourcesResolvers(libs) as IResolvers,
       createNetworkTopNFlowResolvers(libs) as IResolvers,
       createWhoAmIResolvers() as IResolvers,
+      createKpiNetworkResolvers(libs) as IResolvers,
```

Comment:
For context, these `as casts` and methodology is from the Observability (which used to be known as Infra) ... In case you were wondering where we got this pattern. It came from `infra_server.ts`

We are code thieves from sister projects when we can be  :-). Which does mix a bit of different styles and idioms and philosophies but it's good to know where some code came from.